chef/cookbooks/hyperv/recipes/register_services_nova.rb

```diff
@@ -14,5 +14,5 @@
 service "nova-compute" do
   service_name node[:service][:nova][:name]
   action [:enable, :start]
-  subscribes :restart, "template[#{node[:openstack][:config].gsub(/\\/, "/")}/nova.conf]"
+  subscribes :restart, "template[#{node[:openstack][:nova][:config].gsub(/\\/, "/")}/nova.conf]"
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)